sdk/nodejs/randomPassword.ts

```diff
@@ -19,11 +25,16 @@ import * as utilities from "./utilities";
  * 
  * const password = new random.RandomPassword("password", {
  *     length: 16,
- *     overrideSpecial: "/@\" ",
+ *     overrideSpecial: "_%@",
  *     special: true,
  * });
  * const example = new aws.rds.Instance("example", {
- *     password: password.result,
+ *     allocatedStorage: 64,
+ *     engine: "mysql",
+ *     instanceClass: "db.t3.micro",
+ *     password: "",
+ *     "random_string.password.result": [{}],
```

Comment:
This is very strange: what does the source example look like?

Reply:
```hcl
resource "random_password" "password" {
  length = 16
  special = true
  override_special = "_%@"
}

resource "aws_db_instance" "example" {
  instance_class = "db.t3.micro"
  allocated_storage = 64
  engine = "mysql"
  username = "someone"
  password = random_string.password.result
}
```

Reply:
I'd say this looks like a bug in tf2pulumi, @pgavlin?

Reply:
Their example is not correct

```
resource "random_password" "password" {
  length = 16
  special = true
  override_special = "_%@"
}

resource "aws_db_instance" "example" {
  instance_class = "db.t3.micro"
  allocated_storage = 64
  engine = "mysql"
  username = "someone"
  password = random_string.password.result
}
```

Notice it isn't using the `${random_password.password.result}`

Reply:
https://github.com/terraform-providers/terraform-provider-random/pull/77 is partly related here @pgavlin - the source example is wrong - but our handling of it is probably not ideal either.

Reply:
I think that this is a `tf2pulumi` issue. Here's what happens when I try to generate this locally:
```
hela:t pgavlin$ tf2pulumi >index.ts
error: creating tree module: Error parsing main.tf: At 12:14: Unknown token: 12:14 IDENT random_string.password.result
```

Pretty sure that this is valid HCL2 syntax, and that's what `tf2pulumi` is choking on.

Reply:
@pgavlin the password uses `random_string` but the resource is `random_password`

Reply:
Yeah, but that's a semantic error, not a syntax error. `tf2pulumi` can't even parse the file when I run it locally. `tf2pulumi` can tolerate the semantic error; we'd just generate something that references the unknown resource.